packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -196,19 +201,22 @@ class PlatformViewsService {
     return controller;
   }
 
-  // TODO(amirh): reference the iOS plugin API for registering a UIView factory once it lands.
-  /// This is work in progress, not yet ready to be used, and requires a custom engine build. Creates a controller for a new iOS UIView.
+  // TODO(amirh): reference the iOS plugin API for registering a UIView factory
+  // once it lands.
+
+  /// This is work in progress, not yet ready to be used, and requires a custom
```

Comment:
(unrelated, but maybe we can update) This feels horribly out of date. Maybe somebody from the iOS side can advise on how to update this to more accurately reflect reality? (cc @stuartmorgan)

Reply:
cc @cbracken

Reply:
Thanks! I was looking for this exact comment about a week ago! Spotted it about 6 months ago and decided to leave this till we'd decided on what the final UiKitView vs AppKitView API looked like, then went to fix it the other day and couldn't remember the exact wording or location (was looking on the UiKitView class and related render view etc. themselves).

I'll send a quick patch to rewrite this consistently and also fix the TODO while I'm in there.

Reply:
Feel free to land this cleanup first though; probably easier to deal with than the merge conflict from my rewrite!

Reply:
Thanks Chris!

Reply:
Fixed in https://github.com/flutter/flutter/pull/135040